test/interpreter/function-ceiling.math.spec.ts

```diff
@@ -0,0 +1,88 @@
+import {HyperFormula} from '../../src'
+import {ErrorType} from '../../src/Cell'
+import {ErrorMessage} from '../../src/error-message'
+import {adr, detailedError} from '../testUtils'
+
+describe('Function CEILING.MATH', () => {
+  it('should return error for wrong number of arguments', () => {
+    const engine = HyperFormula.buildFromArray([
+      ['=CEILING.MATH()'],
+      ['=CEILING.MATH(1, 2, 3, 4)'],
+    ])
+
+    expect(engine.getCellValue(adr('A1'))).toEqualError(detailedError(ErrorType.NA, ErrorMessage.WrongArgNumber))
+    expect(engine.getCellValue(adr('A2'))).toEqualError(detailedError(ErrorType.NA, ErrorMessage.WrongArgNumber))
+  })
+
+  it('should return error for arguments of wrong type', () => {
+    const engine = HyperFormula.buildFromArray([
+      ['=CEILING.MATH("foo")'],
+      ['=CEILING.MATH(1, "bar")'],
+      ['=CEILING.MATH(1, 2, "baz")'],
+    ])
+
+    expect(engine.getCellValue(adr('A1'))).toEqualError(detailedError(ErrorType.VALUE, ErrorMessage.NumberCoercion))
+    expect(engine.getCellValue(adr('A2'))).toEqualError(detailedError(ErrorType.VALUE, ErrorMessage.NumberCoercion))
+    expect(engine.getCellValue(adr('A3'))).toEqualError(detailedError(ErrorType.VALUE, ErrorMessage.NumberCoercion))
+  })
+
+  it('should work', () => {
+    const engine = HyperFormula.buildFromArray([
+      ['=CEILING.MATH(4.43, 0.3)'],
+      ['=CEILING.MATH(4.43, 0.6)'],
+      ['=CEILING.MATH(4.43, 2)'],
+      ['=CEILING.MATH(4.43)'],
+      ['=CEILING.MATH(-4.43)'],
+      ['=CEILING.MATH(-3.14, -1.8)'],
+      ['=CEILING.MATH(-3.14, 0)'],
+      ['=CEILING.MATH(3.14, 0)'],
+    ])
+
+    expect(engine.getCellValue(adr('A1'))).toEqual(4.5)
+    expect(engine.getCellValue(adr('A2'))).toEqual(4.8)
+    expect(engine.getCellValue(adr('A3'))).toEqual(6)
+    expect(engine.getCellValue(adr('A4'))).toEqual(5)
+    expect(engine.getCellValue(adr('A5'))).toEqual(-4)
+    expect(engine.getCellValue(adr('A6'))).toEqual(-1.8)
+    expect(engine.getCellValue(adr('A7'))).toEqual(0)
+    expect(engine.getCellValue(adr('A8'))).toEqual(0)
+  })
+
+  it('should work with mode for negative numbers', () => {
+    const engine = HyperFormula.buildFromArray([
+      ['=CEILING.MATH(-11, -2)'],
+      ['=CEILING.MATH(-11, -2, 0)'],
+      ['=CEILING.MATH(-11, -2, 1)'],
+      ['=CEILING.MATH(-11, 0, 1)'],
+      ['=CEILING.MATH(-11, 0, 0)'],

```

Comment:
Excel docs are wrong again? "By default, significance is +1 for positive numbers and -1 for negative numbers."

https://support.microsoft.com/en-us/office/ceiling-math-function-80f95d2f-b499-4eee-9f16-f795a8e306c8

Reply:
wording is weird, but it's actually consistent

Reply:
Ok, so they do not mention the actual mode value, just how it will work by default 🤔

Reply:
if you look at our implementation, we actually also set significance to be +1 or -1 depending on input value